### PR TITLE
[Resource] Add missing ref label before timestable interface

### PR DESCRIPTION
**Pull request by tuka217**
| Q             | A
| ------------- | ---
| Doc fix?      | yes
| New docs?  | no
| Fixed tickets    | -

### CODE REVIEW

File: components/Resource/model.rst [hunk 1]
```diff
@@ -17,6 +17,7 @@ This interface will ask you to implement the following methods to your model, th
 | setDeletedAt(\DateTime $deletedAt) | Cet deletion time.                       | Void              |
 +------------------------------------+------------------------------------------+-------------------+
 
+.. _component_resurce_model_timestampable-interface:
```
Comment (TheMadeleine) [inline components/Resource/model.rst]: typo!
``_component_resource_model_timestampable-interface``

**Comment by TheMadeleine** [comment]
@tuka217 can you add to this PR also a ref link to the ``SoftdeletableInterface``?

**Comment by tuka217** [comment]
Done ;)

**Comment by pjedrzejewski** [comment]
Thank you Ania! :)

**Pull request merged**